app/views/vendors/_vendor_row.html.erb

```diff
@@ -1,4 +1,4 @@
-<tr>
+<tr data-vendor-id="<%= vendor_row.id %>">
```

Comment:
Adding a `data` attribute strictly for testing is unnecessary and confusing. You can inspect the output and look at the rows via Nokogiri if needed.